scripts/openzfs-merge.sh

```diff
@@ -103,6 +103,7 @@ generate_desc() {
 	USER_MAIL=$(git config user.email)
 	OPENZFS_COMMIT_AUTHOR=$(git log --format="%aN <%aE>" --remotes=openzfs/master -n 1 $OPENZFS_COMMIT)
 	sed -i '/^$/d' "$TMP_FILE"
+	sed -i '/^Closes #/d' "$TMP_FILE"
```

Comment:
If we're going to automatically remove these lines I'd suggest handling [all keywords](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) which can be used for this.